lib/unexpectedMitm.js

```diff
@@ -730,6 +730,46 @@ module.exports = {
                     return recordedExchanges;
                 });
             })
+            .addAssertion('<any> with http mocked out [and verified] [returning the file] <string> <assertion>', function (expect, subject, filePathWithoutExt) {
```

Comment:
I think requiring the extension would make it easier to add more on-disc formats later.